### DISCUSSION TITLE
PSADT 4.0

### DISCUSSION THREAD

**Discussion by Pollewops** [PSAppDeployToolkit/PSAppDeployToolkit#1065] [General]
When we can expect version 4.0?

Days, weeks, months from now?

**Comment by thangarajua** [comment]
same question. Also, we need to verify that it remains open source and free to use.

**Comment by That-Annoying-Guy** [comment]
[This](https://discourse.psappdeploytoolkit.com/t/info-on-psappdeploytoolkit-v4/5814) says late 2024.

**Comment by mjr4077au** [comment]
I'm satisfied that the blog posts answers enough questions for now. Regarding the specifics around licensing, the project will still be using the LGPL license for v4.
